prov/verbs/src/verbs_ep.c

```diff
@@ -889,6 +891,8 @@ fi_ibv_send_buf_inline(struct fi_ibv_ep *ep, struct ibv_send_wr *wr,
 {
 	struct ibv_sge sge = fi_ibv_init_sge_inline(buf, len);
 
+	assert(wr->wr_id == VERBS_INJECT_FLAG);
```

Comment:
I think you meant !=

Reply:
this function is used for inject send/write. here this assert() checks that flag is set